src/back/TicketManager/src/Core/TicketManager.Core.Services/Migrations/20230420092026_AddTakenSeatsToSector.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace TicketManager.Core.Services.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddTakenSeatsToSector : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<byte[]>(
```

Comment:
We have to do something with those migrations. I don't understand why default value changes those columns all the time. First of all it add useless lines to you PR and secondly it makes migrations much harder to read

Reply:
So do it at least in the future